tests/network-tests/src/Api.ts

```diff
@@ -124,7 +124,10 @@ export class ApiFactory {
 
   constructor(api: ApiPromise, treasuryAccountUri: string, miniSecret: string) {
     this.api = api
-    this.keyring = new Keyring({ type: 'sr25519', ss58Format: JOYSTREAM_ADDRESS_PREFIX })
+    this.keyring = new Keyring({
```

Comment:
Please use git to remove the changes on this file to reduce potential conflicts later